rwengine/src/script/ScriptMachine.hpp

```diff
@@ -92,6 +92,9 @@ struct SCMThread {
     unsigned int stackDepth;
     /// Stores the return-addresses for calls.
     std::array<pc_t, SCM_STACK_DEPTH> calls;
+
+    bool deathOrArrestCheck;
```

Comment:
Shouldn't it and other vars be initialized to false?

Reply:
It is done in ScriptMachine::startThread, see ScriptMachine.cpp

Reply:
Does this get stored in the save files?

Reply:
Yeah, +0x83 and +0x84 bytes